src/nft/components/collection/ActivityCells.tsx

```diff
@@ -188,16 +186,7 @@ const PriceTooltip = ({ price }: { price: string }) => (
 )
 
 export const PriceCell = ({ marketplace, price }: { marketplace?: Markets | string; price?: string | number }) => {
-  const isNftGraphqlEnabled = useNftGraphqlEnabled()
-  const formattedPrice = useMemo(
-    () =>
-      price
-        ? isNftGraphqlEnabled
-          ? formatEth(parseFloat(price?.toString()))
-          : putCommas(formatEthPrice(price.toString()))?.toString()
-        : null,
-    [isNftGraphqlEnabled, price]
-  )
+  const formattedPrice = useMemo(() => (price ? formatEth(parseFloat(price?.toString())) : null), [price])
```

Comment:
Should this be pulled into a util?

Reply:
formatEth is already a util, otherwise we're just making sure price is passed in as a number